author/terraform.tfvars.example

```diff
@@ -0,0 +1,11 @@
+env="xxxx"
```

Comment:
I'm not sure we should set the `env` by default in the file as this needs to change dependent on user / environment as we don't want other developers clashing in our AWS accounts.